src/main/kotlin/com/terraformation/backend/accelerator/db/ReportStore.kt

```diff
@@ -491,6 +494,20 @@ class ReportStore(
             .convertFrom { it.toInt() }
       }
 
+  private val treesPlantedField =
+      with(PLANTINGS) {
+        DSL.field(
+                DSL.select(DSL.sum(NUM_PLANTS))
+                    .from(this)
+                    .join(DELIVERIES)
+                    .on(DELIVERIES.ID.eq(DELIVERY_ID))
+                    .join(WITHDRAWALS)
+                    .on(WITHDRAWALS.ID.eq(DELIVERIES.WITHDRAWAL_ID))
+                    .where(WITHDRAWALS.WITHDRAWN_DATE.between(REPORTS.START_DATE, REPORTS.END_DATE))
+                    .and(plantingSites.PROJECT_ID.eq(REPORTS.PROJECT_ID)))
```

Comment:
Since this query has explicit joins to the deliveries and withdrawals tables, it might be a little clearer to do an explicit join for planting sites too just so we're not mixing explicit and implicit joins in the same query.